desktop/src/scenes/Analyze/analyze.jsx

```diff
@@ -96,7 +96,7 @@ export class Analyze extends Component {
   }
 
   back = () => {
-    this.props.history.push(`/`)
+    window.history.back()
```

Comment:
Shouldn't this depend on whether you are in electron or not? There is an existing `isElectron` function you can use for this.

Reply:
tested this in the desktop app and it works fine